temba/orgs/models.py

```diff
@@ -1347,10 +1347,8 @@ def get_low_credits_threshold(self):
 
     def _calculate_low_credits_threshold(self):
         now = timezone.now()
-        unexpired_topups = self.topups.filter(is_active=True, expires_on__gte=now)
-
-        active_topup_credits = [topup.credits for topup in unexpired_topups if topup.get_remaining() > 0]
-        last_topup_credits = sum(active_topup_credits)
+        last_topup = self.topups.filter(is_active=True).order_by("created_on").last()
```

Comment:
Sorry, the description of the ticket is not totally correct, I've commented on Asana ticket about this https://app.asana.com/0/1159876841343611/1171619451194923/f

Reply:
@ihor-palii - Since you have implemented the solution on `has_low_credits`, could you remove those changes? Thanks

Reply:
Sure. I've reverted commit with changes.